<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Components Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="categoryFilter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Category</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="typeFilter">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Type</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="componentFilter">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Component</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="valueFilter">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Value</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="packageFilter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Package</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConfiguration"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionFrom_csv"/>
     <addaction name="actionFrom_xls"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionTo_csv"/>
     <addaction name="actionTo_xls"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="menuExport"/>
    <addaction name="actionRestore_from_db_file"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuComponents">
    <property name="title">
     <string>Components</string>
    </property>
    <addaction name="actionAdd_new"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocs"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuComponents"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAdd_new">
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDocs">
   <property name="text">
    <string>Docs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRestore_from_db_file">
   <property name="text">
    <string>Restore from *.db file</string>
   </property>
  </action>
  <action name="actionFrom_csv">
   <property name="text">
    <string>From *.csv</string>
   </property>
  </action>
  <action name="actionFrom_xls">
   <property name="text">
    <string>From *.xml</string>
   </property>
  </action>
  <action name="actionTo_csv">
   <property name="text">
    <string>To *.csv</string>
   </property>
  </action>
  <action name="actionTo_xls">
   <property name="text">
    <string>To *.xml</string>
   </property>
  </action>
  <action name="actionDeleteComponent">
   <property name="text">
    <string>Delete Component</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionIncreaseTheState">
   <property name="text">
    <string>Increas the state</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionIncreaseTheStateBy">
   <property name="text">
    <string>Increase the state by...</string>
   </property>
  </action>
  <action name="actionResetState">
   <property name="text">
    <string>Reset state</string>
   </property>
  </action>
  <action name="actionReduceTheStateBy">
   <property name="text">
    <string>Reduce the state by ...</string>
   </property>
  </action>
  <action name="actionReduceTheState">
   <property name="text">
    <string>Reduce the state</string>
   </property>
  </action>
  <action name="actionEditComponent">
   <property name="text">
    <string>Edit component</string>
   </property>
  </action>
  <action name="actionUpdateDatabase">
   <property name="text">
    <string>Update Database</string>
   </property>
  </action>
  <action name="actionLookForDocumentationAtTMEeu">
   <property name="text">
    <string>Look for documentation at TME.eu</string>
   </property>
  </action>
  <action name="actionLookForDocumentationAtMouserCom">
   <property name="text">
    <string>Look for documentation at Mouser.com</string>
   </property>
  </action>
  <action name="actionLookForDocumentationAtDigiKeyCom">
   <property name="text">
    <string>Look for documentation at DigiKey.com</string>
   </property>
  </action>
  <action name="actionLookForDocumentationAtFarnellCom">
   <property name="text">
    <string>Look for documentation at Farnell.com</string>
   </property>
  </action>
  <action name="actionLookForDocumentationAtAllDatasheetCom">
   <property name="text">
    <string>Look for documentation at AllDatasheet.com</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
